### PR TITLE
Export layer defaults and expand read blacklist

### PR DESCRIPTION
**Pull request by vincentmvdm**
## Changes
- Export layer defaults that non-Figma environments like tests/APIs can use to create figma-json. Update by clicking the plugin's "Log defaults" button and copying the output.
- Improve read blacklist by adding a few new properties

## Test plan
- Tested that clicking the "Log defaults" button logs the default layer representations to the console
- Verified that reading/inserting works like before
- Ran the test suite

### CODE REVIEW

File: src/figma-default-layers.ts [hunk 1]
```diff
@@ -0,0 +1,604 @@
...
+  RECTANGLE: F.RectangleNode;
+  LINE: F.LineNode;
+  ELLIPSE: F.EllipseNode;
+  POLYGON: F.PolygonNode;
+  STAR: F.StarNode;
+  VECTOR: F.VectorNode;
+  TEXT: F.TextNode;
+  FRAME: F.FrameNode;
+} = {
+  RECTANGLE: {
```
Comment (darknoon) [inline src/figma-default-layers.ts:15]: This might be slightly nicer as individual exports, since you can always grab all of them via `import *`, then just grab what you need righ?

File: src/figma-default-layers.ts [hunk 2]
```diff
@@ -0,0 +1,604 @@
...
+    reactions: [],
+    type: "ELLIPSE"
+  },
+  POLYGON: {
+    id: "_",
+    name: "Polygon",
+    removed: false,
+    visible: true,
+    locked: false,
+    componentPropertyReferences: null,
```
Comment (darknoon) [inline src/figma-default-layers.ts:224]: hmm, why is this here?

**Review by darknoon — APPROVED** [review]
Seems good. I do wonder if we're missing some utility from not having that file be JSON, eg for use with python.

File: src/figma-default-layers.ts (hunk 2 above)
Comment (vincentmvdm) [reply src/figma-default-layers.ts:224]: Extra info that might prove useful in the future: https://www.figma.com/plugin-docs/api/node-properties/#componentpropertyreferences. We won't need it depending on how we'll handle components props but left it in in case we do.It comes by default with Figma layers and shouldn't be actively harmful

File: src/figma-default-layers.ts (hunk 1 above)
Comment (vincentmvdm) [reply src/figma-default-layers.ts:15]: Good point, noted.

File: src/figma-default-layers.ts (hunk 2 above)
Comment (darknoon) [reply src/figma-default-layers.ts:224]: Interesting. Wondering if we should keep that modeled on the component parent or the sublayer. fine for now.
